patches/server/0830-Flying-Fall-Damage.patch

```diff
@@ -0,0 +1,85 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TreyRuffy <treyruffy@users.noreply.github.com>
+Date: Fri, 10 Dec 2021 20:23:43 -0700
+Subject: [PATCH] Flying Fall Damage
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index cd918cec00d8202252af0d20b1a8891371c538e3..7f3c9607a35806b7002a8a86a6c99942274fc41b 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -118,6 +118,11 @@ public class PaperConfig {
+         enableBrigadierConsoleCompletions = getBoolean("settings.console.enable-brigadier-completions", enableBrigadierConsoleCompletions);
+     }
+ 
++    public static boolean enableFlyingFallDamage = false;
++    private static void flyingFallDamage() {
++        enableFlyingFallDamage = getBoolean("settings.enable-default-flying-fall-damage", enableFlyingFallDamage);
++    }
++
+     public static void registerCommands() {
+         for (Map.Entry<String, Command> entry : commands.entrySet()) {
+             MinecraftServer.getServer().server.getCommandMap().register(entry.getKey(), "Paper", entry.getValue());
+diff --git a/src/main/java/net/minecraft/world/entity/player/Player.java b/src/main/java/net/minecraft/world/entity/player/Player.java
+index 2c00a766130a7f682fc6c4c74321e10637ca7932..b335773eee42ff17aab98c6f1df87b38cddeaab3 100644
+--- a/src/main/java/net/minecraft/world/entity/player/Player.java
++++ b/src/main/java/net/minecraft/world/entity/player/Player.java
+@@ -179,6 +179,7 @@ public abstract class Player extends LivingEntity {
+     public FishingHook fishing;
+     // Paper start
+     public boolean affectsSpawning = true;
++    public boolean flyingFallDamage = com.destroystokyo.paper.PaperConfig.enableFlyingFallDamage;
```

Comment:
I think that this should be a tristate, where if UNSET it will not be saved to the player entity